rets/client/decoder.py

```diff
@@ -31,8 +32,16 @@ def decode_field(field: str, value: str) -> Any:
                 return None
             return decoders[field](value)
 
-        return tuple(OrderedDict((field, decode_field(field, value)) for field, value in row.items())
```

Comment:
this is where the error happens. e.g. https://honeycomb-airflow-production.managed.services.opendoor.com/admin/airflow/log?task_id=mls-ingestion-carets_property_residentialproperty&dag_id=dwellings-mls-ingest-carets_property_residentialproperty&execution_date=2019-05-12T14%3A00%3A00%2B00%3A00